teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,58 +4,83 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe start with sth more in the direction of the below quote: understanding protein–ligand interactions which are driving molecular recognition are fundamental in (rational) drug design.</p><p>Then that we use PLIP (and NGLView) to do so (maybe link PLIP already here?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>